src/hypervisor.adoc

```diff
@@ -546,10 +546,14 @@ read-only in `hip`, and is the logical-OR of these interrupt sources:
 * any other platform-specific external interrupt signal directed to
 VS-level.
 
-Bits `hip`.VSTIP and `hie`.VSTIE are the interrupt-pending and
-interrupt-enable bits for VS-level timer interrupts. VSTIP is read-only
-in `hip`, and is the logical-OR of `hvip`.VSTIP and any other
-platform-specific timer interrupt signal directed to VS-level.
+Bits `hip`.VSTIP and `hie`.VSTIE are the interrupt-pending and interrupt-enable
+bits for VS-level timer interrupts. VSTIP is read-only in `hip`, and is the
+logical-OR of `hvip`.VSTIP and the timer interrupt signal resulting from
+`vstimecmp` (if `vstimecmp` is implemented). The `hip`.VSTIP bit, in response to
+timer interrupts generated by `vstimecmp`, is set and cleared by writing
+`vstimecmp` with a value that respectively is less than or equal to, or greater
```

Comment:
I realize this is old text, but it is hard to parse, and we might as well fix it up while we're here.  Maybe try something like:

```
The `hip`.VSTIP bit, in response to timer interrupts generated by `vstimecmp`, is set by writing `vstimecmp` with a value that is less than or equal to the sum of `time` and `htimedelta`, truncated to 64 bits; it is cleared by writing `vstimecmp` with a greater value.
```

Reply:
updated.